python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -52,7 +52,7 @@ def _get_node_name(node_info: Mapping[str, Any]):
 
 
 def _get_node_asset_key(node_info):
-    return AssetKey(node_info["name"])
+    return AssetKey([node_info["schema"], node_info["name"]])
```

Comment:
I'm not 100% sure that schema will always be available here. if you are, go ahead, otherwise, making this a filter(None, [node_info.get("schema"), node_info["name"]) would be great.

Reply:
I think you are right that we need to consider schema not being available